packages/subgraph/src/packages/token/governance-erc20.ts

```diff
@@ -79,8 +79,15 @@ export function handleDelegateVotesChanged(event: DelegateVotesChanged): void {
 
   if (event.params.delegate != Address.zero()) {
     let member = getOrCreateMember(event.params.delegate, pluginId);
-    // Assign the cumulative delegated votes to this member from all their delegators.
-    member.votingPower = event.params.newBalance;
-    member.save();
+    if (
+      member.balance.le(BigInt.zero()) &&
```

Comment:
why `le` and not `eq`?

Reply:
I used `le` just in case, but the value should never be lower than 0. I can change it to `eq` if you prefer it

Reply:
yea, please use eq